assets/scss/editor.scss

```diff
@@ -1,4 +1,73 @@
 $govuk-assets-path: '/wp-content/themes/govuk-theme/static/lib/govuk-frontend/govuk/assets/';
 
+/*--------------------------------------------------------------
+ # Settings
+
+ Define our global preprocessor variables and CSS custom 
+ properties.
+--------------------------------------------------------------*/
+@import 'settings/settings';
+
+
+/*--------------------------------------------------------------
+ # Helpers
+
+ Functions and mixins.
+--------------------------------------------------------------*/
+@import 'helpers/aspect-ratio';
+@import 'helpers/colour';
+@import 'helpers/font-faces';
+@import 'helpers/typography';
+
+
+/*--------------------------------------------------------------
+ # GOV.UK Frontend
+
+ Use the framework's starter styles and make helper mixins and 
+ functions available.
+--------------------------------------------------------------*/
 @import "node_modules/govuk-frontend/govuk/all";
+
+
+/*--------------------------------------------------------------
+ # Base
```

Comment:
Do we want to call this "base", or "core", which would seem to align more with the govuk-frontend naming conventions?

Reply:
That makes sense and is true to the purpose of this PR, thanks. Change made.